benchmark/opperf/utils/benchmark_utils.py

```diff
@@ -148,6 +148,11 @@ def run_op_benchmarks(ops, dtype, ctx, profiler, warmup, runs):
     for op, op_params in ops.items():
         # Prepare inputs for the operator
         inputs = prepare_op_inputs(op, op_params)
+
+        # Testing Embedding op backwards is currently unsupported
+        if op == "Embedding":
```

Comment:
Also just adding in `run_op_benchmarks` doesn't seal all ends
There are couple of ways to run OpPerf
For eg what happens when run_performance_test is used?
Does he get an error? or will it seg_fault?

Reply:
When `run_performance_test` is used on Embedding with `run_backward=True`, an error will be thrown. I can document this here as well if you would like.

Reply:
Yes please do.

Reply:
Done! :)